playbooks/roles/document_hosting_k8s/tasks/main.yaml

```diff
@@ -0,0 +1,207 @@
+---
+- name: Create Document Hosting Namespace
+  community.kubernetes.k8s:
+    context: "{{ context }}"
+    state: "{{ state }}"
+    name: "{{ namespace }}"
+    api_version: "v1"
+    kind: "Namespace"
+
+
+- name: Create Document Hosting Service Account
```

Comment:
Do we really need separate SA? I do not see any reason so far.

Reply:
we require (in openshift) that the serviceaccount has privileged scc assigned. generally it is better to have a specific SA per service (instead of using the default)...however in our case we use separate namespace where we are supposed to run only one app...so from this point of view perhpas assigning privileged scc to default would also work.